compatibility with ghc-8 rc2

aavogt opened the pull request.

aavogt committed:
compatibility with ghc-8 rc2

Icelandjack commented:
Duplicate of #7, oops

domenkozar commented:
cc @bmillwood

bmillwood commented:
I don't maintain this package anymore. @Icelandjack you're on the collaborators list. Do you want to take it over?

aavogt committed:
Treat (a,b) like (,) a b, and likewise for [a,b]

   Icelandjack's bug https://github.com/bmillwood/applicative-quoters/issues/8

   It is possible that [i|(a,b,c)|] was written instead of
   (pure(a,b,c)), but given that the quasiquote-free solution is easier
   to understand, more reliable and only slightly longer it is unlikely
   to cause problems.
